gocd/sp.target.gocd.yaml

```diff
@@ -151,3 +167,49 @@ pipelines:
           - staging-bot
         tasks:
           - script: osc -A https://api.suse.de release SUSE:SLE-15-SP5:GA:TEST
+
+  SLE15.SP5.ReleaseScc:
+    parameters:
+      product: sle15sp5
+      target: MyTarget
+      build: "0.0"
+    materials:
+      scripts:
+        auto_update: true
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+        whitelist:
+          - DO_NOT_TRIGGER
+        destination: scripts
+    stages:
+    - Release.Images.Update:
+        approval: manual
+        roles:
+          - SLE
+        resources:
+          - staging-bot
+        tasks:
+          - script: curl --verbose --max-time 3600 "http://10.160.0.10:5000/main/update?product=#{product}&target=#{target}&build=#{build}"
```

Comment:
I'd expect that this needs `-X POST` or something, `GET` requests must not change state

Reply:
I am aware. Thanks for the hint.